locales/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "frontpage_documentTitle": "Data Marketplace",
+  "frontpage_popularProductsHeader": "Popular"
+}
```

Comment:
I would like for us to be able to namespace translations. I see this getting cumbersome to maintain at scale >.<

Reply:
By changing library or by using the IntlProvider in the pages instead? Might perhaps be easier with another library 🤔 If we at some point want to remove the string to outside of the code base, we can solve that problem when it arises

Reply:
unless into-react has a limitation, couldn't we namespace by file?